pkg/util/selector.go

```diff
@@ -146,6 +146,9 @@ func ResourceMatchSelectors(resource *unstructured.Unstructured, selectors ...po
 // ResourceMatchSelectorsPriority returns the highest priority between specific resource and the selectors.
 func ResourceMatchSelectorsPriority(resource *unstructured.Unstructured, selectors ...policyv1alpha1.ResourceSelector) ImplicitPriority {
 	var priority ImplicitPriority
+	if len(selectors) == 0 {
```

Comment:
For pp and cpp, it is correct here? I suppose an empty selector does not select any pp and cpp here.

Reply:
> I suppose an empty selector does not select any pp and cpp here.

I agree with you.
As I said, https://github.com/karmada-io/karmada/pull/2609#discussion_r1033527301

but, 
https://github.com/karmada-io/karmada/pull/2609#discussion_r1033422114 and https://github.com/karmada-io/karmada/pull/2609#discussion_r1034233008

Reply:
@chaunceyjiang Can you summary the answer here?

Reply:
the function `ResourceMatchSelectorsPriority` default return to `MatchAll` when selectors is nil.
@jwcesign @XiShanYongYe-Chang  


the function `ResourceMatchSelectorsPriority` default return to `MisMatch` when selectors is nil.
@chaunceyjiang @Garrybest

Reply:
I prefer to return `MisMatch ` by default